Assets/Tayx/Graphy - Ultimate Stats Monitor/Scripts/Dummy/SODummy.cs

```diff
@@ -0,0 +1,9 @@
+﻿using UnityEngine;
+
+namespace Tayx.Graphy
+{
+    public class SODummy : ScriptableObject
```

Comment:
Can you rename the class to have G_ as a prefix? That way it's consistent with the rest of the code :)